src/_utils/helpers.js

```diff
@@ -8,3 +9,65 @@ export const isCountCutoffExceeded = (cutoff = 100) => (pageNumber, limit) => {
 };
 
 export const fetchData = url => axios.get(url);
+
+const MAP_QUERY_ARG = {
+  status: 'current_status',
+  'Model Numbers': 'model_number',
+  'Asset Types': 'asset_type',
+  Cohort: 'cohort',
+  'Asset Assigned': 'asset_count'
+};
+
+const buildQueryString = (key = '', data = {}) => `&${MAP_QUERY_ARG[key]}=${data[key]}`;
+
+/**
+ * filters = {
+ *   Cohort: 12,
+ *   'Model Numbers': 123,
+ *   'Asset Assigned': 3
+ * }
+ *
+ * @param {object} filters
+ */
+const buildFiltersQueryString = (filters) => {
+  let query = '';
+  Object.keys(filters).forEach((filterType) => {
+    query += buildQueryString(filterType, filters);
+  });
+  return query;
+};
+
+/**
+ * This method is a helper function created to help with creating
+ * dynamic API endpoints, together with the query strings.
+ *
+ * optionalArgs is an object of the form:
+ * {
+ *   filters: {},
+ *   status: ''
+ * }
+ * You can add more to the object as the need arises.
+ *
+ * @param {string} entity the API endpoint such as `/manage-assets`
+ * @param {number} pageNumber
+ * @param {number} pageSize limit per page
+ * @param  {object} optionalArgs holds optional parameters like filters, status, etc, as an object
+ */
+export const constructApiUrl = (entity, pageNumber, pageSize, optionalArgs = {}) => {
```

Comment:
This will be the method to use in future logic to create the API endpoint together with the query strings. After racking my brain for a while, I came up with this version as it is dynamic enough to work for all endpoints we are currently using.